README.md

```diff
@@ -19,7 +19,7 @@ This component is responsible for taking submissions data gathered by [uptime-se
 
 1. **Install dependencies:**
 
-```sh
+```bash
```

Comment:
I think `sh` is okay here, as it denotes code suitable for any POSIX shell, not just bash. There are no bashisms here that I can see, so why not `sh`? Does it not work?

Reply:
I think I just did it for consistency across readme, it might as well be nothing, but fair enough.